ext/AvizExt/viz/viz.jl

```diff
@@ -1,7 +1,7 @@
 using Makie, DataFrames
 
 using ADRIA: ResultSet, metrics.metric_label, analysis.col_normalize, model_spec
-using NamedDims, AxisKeys
+using NamedDims, AxisKeys, YAXArrays, DimensionalData
```

Comment:
Is the import of DimensionalData simply to have the `At` method?

If so, I'd prefer it be imported via YAXArrays:

```julia
import YAXArrays.DD: At
```

Reply:
I think I just took this from the YAXArray docs and forgot to remove, I've removed it now